app/javascript/dashboard/routes/dashboard/settings/sla/AddSLA.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="h-auto overflow-auto flex flex-col">
+    <woot-modal-header
+      :header-title="$t('SLA.ADD.TITLE')"
+      :header-content="$t('SLA.ADD.DESC')"
+    />
+    <form class="mx-0 flex flex-wrap" @submit.prevent="addSLA">
+      <woot-input
+        v-model.trim="name"
+        :class="{ error: $v.name.$error }"
+        class="w-full"
+        :label="$t('SLA.FORM.NAME.LABEL')"
+        :placeholder="$t('SLA.FORM.NAME.PLACEHOLDER')"
+        :error="getSlaNameErrorMessage"
+        data-testid="sla-name"
+        @input="$v.name.$touch"
+      />
+
+      <woot-input
+        v-model.trim="description"
+        class="w-full"
+        :label="$t('SLA.FORM.DESCRIPTION.LABEL')"
+        :placeholder="$t('SLA.FORM.DESCRIPTION.PLACEHOLDER')"
+        data-testid="sla-description"
+      />
+
+      <woot-input
+        v-model.trim="firstResponseTimeThreshold"
```

Comment:
I would suggest combining this and the edit forms together as it is the same UI with the same fields. For reference, please check WebhookForm component.

Reply:
done